src/game/mod.rs

```diff
@@ -0,0 +1,227 @@
+use std::collections::HashMap;
+
+use crate::conways_error::ConwaysError;
+use crate::cell::Cell;
+
+pub struct Game {
+    live_cells: Vec<Cell>,
+}
+
+impl Game {
+    pub fn new_with_seed(seed: Vec<(i32,i32)>) -> Self {
+        let live_cells = seed.iter().map(|(x,y)| Cell::new(*x,*y)).collect();
+
+        Game {
+            live_cells
+        }
+    }
+
+    pub fn new() -> Self {
+        let live_cells = vec![
+            Cell::new(1, 3),
+            Cell::new(1, 2),
+            Cell::new(1, 1),
+        ];
+
+        Game {
+            live_cells
+        }
+    }
+
+    // Returns the positions of the 8 neighbours of a cell
+    fn neighbour_positions(&self, cell: &Cell) -> Vec<Cell> {
```

Comment:
I like this function, but it could make more sense being in the `Cell` struct

Reply:
Done! This looks so much better now